xarray/tests/test_backends.py

```diff
@@ -1147,9 +1147,11 @@ def test_chunk_encoding_with_dask(self):
 
         # should fail if dask_chunks are irregular...
         ds_chunk_irreg = ds.chunk({'x': (5, 4, 3)})
-        with pytest.raises(ValueError):
+        with pytest.raises(ValueError) as e_info:
```

Comment:
It looks like ` as e_info` was added for debugging but isn't actually necessary?

Reply:
Actually, this is to debug that the correct error is raised. There is a `match` check below.